modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java

```diff
@@ -56,7 +56,7 @@
      */
     public BrowserWebDriverContainer() {
         this.waitStrategy = new LogMessageWaitStrategy()
-                .withRegEx(".*RemoteWebDriver instances should connect to.*\n")
+                .withRegEx(".*Selenium Server is up and running.*\n")
```

Comment:
Is this String available in all versions? Because I thought you wanted something like
```
".*(RemoteWebDriver instances should connect to)|(Selenium Server is up and running).*\n"
```